sensuctl crash after upgrade to survey package

```
? Metric Format:  [Use arrows to move, type to filter]
> none
  nagios_perfdata
  graphite_plaintext
  opentsdb_line
  influxdb_line
  prometheus_text
panic: interface conversion: interface {} is core.OptionAnswer, not string

goroutine 1 [running]:
github.com/sensu/sensu-go/cli/commands/check.(*checkOpts).administerQuestionnaire.func3(0x10e8dc0, 0xc0002cc000, 0x0, 0x0)
	/home/eric/go/src/github.com/sensu/sensu-go/cli/commands/check/interactive.go:237 +0xe5
github.com/AlecAivazis/survey/v2.Ask(0xc00016a1b0, 0x12, 0x12, 0x10df900, 0xc0001650e0, 0x0, 0x0, 0x0, 0xc00029dc28, 0xe99f7260)
	/home/eric/go/pkg/mod/github.com/!alec!aivazis/survey/v2@v2.2.14/survey.go:327 +0x3e9
github.com/sensu/sensu-go/cli/commands/check.(*checkOpts).administerQuestionnaire(0xc0001650e0, 0xc000502400, 0x7, 0x1a1f801)
	/home/eric/go/src/github.com/sensu/sensu-go/cli/commands/check/interactive.go:266 +0xe95
github.com/sensu/sensu-go/cli/commands/check.CreateCommand.func2(0xc00017cf00, 0xc000381330, 0x0, 0x1, 0x0, 0x0)
	/home/eric/go/src/github.com/sensu/sensu-go/cli/commands/check/create.go:44 +0x15e
github.com/spf13/cobra.(*Command).execute(0xc00017cf00, 0xc000381320, 0x1, 0x1, 0xc00017cf00, 0xc000381320)
	/home/eric/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:852 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0xc000131900, 0xc000148780, 0xc00004c0b8, 0x0)
	/home/eric/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/home/eric/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
main.main()
	/home/eric/go/src/github.com/sensu/sensu-go/cmd/sensuctl/main.go:24 +0xbb
```

--- a/cli/commands/check/interactive.go
+++ b/cli/commands/check/interactive.go
@@ -139,8 +139,8 @@ func (opts *checkOpts) administerQuestionnaire(editing bool) error {
 				Default: opts.Cron,
 			},
 			Validate: func(val interface{}) error {
-				if val.(string) != "" {
-					if _, err := cron.ParseStandard(val.(string)); err != nil {
+				if value, ok := val.(string); ok && value != "" {
+					if _, err := cron.ParseStandard(value); err != nil {
 						return err
 					}
 				}
@@ -192,8 +192,11 @@ func (opts *checkOpts) administerQuestionnaire(editing bool) error {
 				Help:    "If check requests are published for the check. Value must be true or false.",
 			},
 			Validate: func(val interface{}) error {
-				_, err := strconv.ParseBool(val.(string))
-				return err
+				if value, ok := val.(string); ok {
+					_, err := strconv.ParseBool(value)
+					return err
+				}
+				return nil
 			},
 		},
 		{
@@ -234,8 +237,8 @@ func (opts *checkOpts) administerQuestionnaire(editing bool) error {
 				Default: opts.OutputMetricFormat,
 			},
 			Validate: func(val interface{}) error {
-				if value := strings.TrimSpace(val.(string)); value != "" && value != "none" {
-					if err := types.ValidateOutputMetricFormat(val.(string)); err != nil {
+				if value, _ := val.(string); value != "" && strings.TrimSpace(value) != "none" {
+					if err := types.ValidateOutputMetricFormat(value); err != nil {
 						return err
 					}
 				}
@@ -257,8 +260,11 @@ func (opts *checkOpts) administerQuestionnaire(editing bool) error {
 				Help:    "if true, schedule this check in a round-robin fashion",
 			},
 			Validate: func(val interface{}) error {
-				_, err := strconv.ParseBool(val.(string))
-				return err
+				if value, ok := val.(string); ok {
+					_, err := strconv.ParseBool(value)
+					return err
+				}
+				return nil
 			},
 		},
 	}...)

--- a/cli/commands/hook/interactive.go
+++ b/cli/commands/hook/interactive.go
@@ -101,7 +101,7 @@ func (opts *hookOpts) administerQuestionnaire(editing bool) error {
 				Default: opts.Stdin,
 			},
 			Validate: func(val interface{}) error {
-				if str := val.(string); str != "false" && str != "true" {
+				if str, ok := val.(string); ok && str != "false" && str != "true" {
 					return fmt.Errorf("Please enter either true or false")
 				}
 				return nil

--- a/cli/commands/silenced/interactive.go
+++ b/cli/commands/silenced/interactive.go
@@ -104,8 +104,11 @@ func (o *silencedOpts) administerQuestionnaire(editing bool) error {
 				Help:    "Start silencing events at this time. Format: Jan 02 2006 3:04PM MST",
 			},
 			Validate: func(val interface{}) error {
-				_, err := timeutil.ConvertToUnix(val.(string))
-				return err
+				if value, ok := val.(string); ok {
+					_, err := timeutil.ConvertToUnix(value)
+					return err
+				}
+				return nil
 			},
 		},
 		{

